### DISCUSSION TITLE
Set Pose Net ONNX question

### DISCUSSION THREAD

**Discussion by tocqueville2001** [opentrack/opentrack#1838] [Q&A]
Hi,

I am attempting to fine-tune my Opentrack, and I have a question about the ONNX files. What exactly are the differences between the following configuration files?

I assume:
- 'BIG' is optimized for larger inputs, potentially offering higher accuracy but requiring more computational resources.
- 'Small' is optimized for efficiency and speed, making it suitable for resource-constrained environments.

head-pose-0.2-big.onnx
head-pose-0.2-small.onnx
head-localizer.onnx
head-pose.onnx

**Comment by YosypC** [comment]
I'd like to have an answer too.

**Comment by DaWelter** [comment]
Oh sure. Must have missed this post.
* head-pose-0.2-big.onnx: Estimates the head pose. Second release. A bigger and more accurate network. Runs slower than the small version. The inputs are the same across all (pose detection) network so far.
* head-pose-0.2-small.onnx: As you suspected. It tends to be noticeably more noisy.
* head-localizer.onnx: Localizes your face. It generates a bounding box around which a crop is taken which is fed into the pose networks.
* head-pose.onnx: The first released model. It should be surpassed in terms of accuracy by the 0.2-big network at least. Only left there in case the 0.2 versions don't work for you.

**Reply by darmach** [reply]
How about those 0.4 versions?

In general I am having problems with stability / jitter. Which is most precise? (I'm aware that it might pick up micro-movements which I will have to probably deadzone/smooth anyway)

Oh, and while at it, maybe I could bother you with one more question - how does the resolution matter? It's probably downscaled when sent for analysis, is it better to have 1080p or should I drop it lower?
